packages/react-docsite-components/src/components/Markdown/Markdown.tsx

```diff
@@ -1,5 +1,5 @@
 import * as React from 'react';
-import MarkdownToJsx, { MarkdownProps as MarkdownToJsxProps } from 'markdown-to-jsx';
+import MarkdownComponent, { MarkdownToJSX } from 'markdown-to-jsx';
```

Comment:
MarkdownToJSX is now the name of an exported namespace containing all the types. The default export is the component itself, which is not included in the namespace.